### DISCUSSION TITLE
bundletool .jar issue

### DISCUSSION THREAD

**Discussion by tashsuga** [appium/appium#16557] [Q&A]
Hello All,

I faced PATH env. issue  even if I correctly set the PATH .  If you have any clues, please let me know.

- selenium-webdriver is Ver. 3.142.7
- appium_lib is Ver. 11.2.0
- appium_lib_core is Ver. 4.7.1
- ruby 3.0.2p107 
- Windows 10 
- bundletool-all-1.8.2.jar
   ==> I changed the file name to "bundletool.jar" in PATH.
------------------------------

_C:/Ruby30-x64/lib/ruby/gems/3.0.0/gems/selenium-webdriver-3.142.7/lib/selenium/webdriver/remote/response.rb:72:in `assert_ok'
    at asyncHandler (/usr/lib/node_modules/appium/node_modules/appium-base-driver/lib/protocol/protocol.js:388:37)
    at getResponseForW3CError (/usr/lib/node_modules/appium/node_modules/appium-base-driver/lib/protocol/errors.js:804:9)
UnknownError: An unknown server-side error occurred while processing the command. Original error: bundletool.jar binary is expected to be present in PATH. Visit https://github.com/google/bundletool for more details.
Error: test_logcat_test1(ContactsAndroidTests): Selenium::WebDriver::Error::UnknownError: An unknown server-side error occurred while processing the command. Original error: bundletool.jar binary is expected to be present in PATH. Visit https://github.com/google/bundletool for more details._

------------------------------

I configured the below, but it was not correctly working.
https://programmerah.com/the-appium-doctor-reports-an-error-bundletool-jar-under-win10-45189/

**Comment by KazuCocoa** [comment]
What is the result of `$Env:Path`? The result was expected to the bundletool.jar?

https://discuss.appium.io/ also helps to ask questions.
